internal/cloud/realm/app.go

```diff
@@ -0,0 +1,74 @@
+package realm
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/10gen/realm-cli/internal/utils/api"
+)
+
+// App represents basic Realm App data
+type App struct {
+	ID          string `json:"_id"`
+	GroupID     string `json:"group_id"`
+	ClientAppID string `json:"client_app_id"`
+	Name        string `json:"name"`
+}
+
+var (
+	errGroupNotFound = errors.New("group could not be found")
+)
+
+var (
+	appsPathPattern = adminAPI + "/groups/%s/apps"
+)
+
+func (c *client) GetAppsForUser() ([]App, error) {
+	profile, err := c.GetAuthProfile()
+	if err != nil {
+		return nil, err
+	}
+	groupIDs := profile.AllGroupIDs()
+	var appArr []App
+	appArr = make([]App, 0)
```

Comment:
this line isn't needed, you can immediately start appending to `var appArr []App` as is

also a nit, let's just call this `arr`, the type and context of this function makes `app` a bit redundant